ci-operator/step-registry/openshift-extended/test/disasterrecovery/openshift-extended-test-disasterrecovery-commands.sh

```diff
@@ -207,6 +207,8 @@ openstack*)
 ibmcloud)
     export TEST_PROVIDER='{"type":"ibmcloud"}'
     IC_API_KEY="$(< "${CLUSTER_PROFILE_DIR}/ibmcloud-api-key")"
+    # Add IBM secret for dr test cases
+    oc create secret generic qe-ibmcloud-creds --from-literal=apiKey=${IC_API_KEY} -n kube-system
```

Comment:
@wangke19 it is risk to cause key leaked even you said there is no ` set -x`.
we can not ensure someone add it in someday. even it is leaked once, it is not accepted.
so, we have to find way to avoid it.
and more, in order to make it unexpected quit, add `|| true`


could you please change it to 
`oc create secret generic qe-ibmcloud-creds --from-file=apiKey="${CLUSTER_PROFILE_DIR}/ibmcloud-api-key" -n kube-system || true`?

Reply:
If failed to create secret `qe-ibmcloud-creds`, we want to get the error earlier, DR cases on IBM clould will run into failure without it.

Reply:
if you all cases depend on this secret, you could get step failed here. 
if not all your cases depends on this secret, some of case are not executed for your such code. while these cases should be executed.

so, you could determine it per the above rule.

if finally you need to get failed at here, I suggest to add more log so that you will know if it fails here.
the suggest code:
```shell
    if [ -f "${SHARED_DIR}/ibmcloud-min-permission-api-key" ]; then
      IC_API_KEY="$(< "${SHARED_DIR}/ibmcloud-min-permission-api-key")"
      echo "using the specified key for minimal permission!!"
      oc create secret generic qe-ibmcloud-creds --from-file=apiKey="${SHARED_DIR}/ibmcloud-min-permission-api-key" -n kube-system
    else
      IC_API_KEY="$(< "${CLUSTER_PROFILE_DIR}/ibmcloud-api-key")"
      echo "try to create secret with ibmcloud-api-key"
      oc create secret generic qe-ibmcloud-creds --from-file=apiKey="${CLUSTER_PROFILE_DIR}/ibmcloud-api-key" -n kube-system
    fi
   echo "secret is created succesully"
```

Reply:
Yes, for apiserve DR cases, if there is no secret `qe-ibmcloud-creds `, cases will run into the failure ` error from server (NotFound): secrets "qe-ibmcloud-creds" not found`.

Reply:
as we talked, there are other modules DR cases which do not depend on this secret.
so, better to add `|| true` to avoid step failing here.

Reply:
Updated.